marketing_emails/email_kickoff_keynote.md

```diff
@@ -0,0 +1,39 @@
+*Fellow Pythonistas!*
+
+It is my honor to announce that PyTN 2018 is on its way.  The 2018 event will take place on February 10th and 11th at the Nashville Law School.  The organizers are working on setting up the 2018 event to meet the level of excellence former chairman Jason Myers set.
+
+We can first announce the keynote speakers who are sure to give you reason enough to put PyTN 2018 on your calendar:
+
+
+#### todo: change for 2018 keynotes
+- Sophie Rapoport: A software engineer from Nashville who has made a distinct impression on me with her sharp intellect, dedication to learning, and teaching nature. Simply an amazing person.
+
+- Courey Elliott: A software developer who I first meet about 2 years ago with an amazing capacity for reasoning, a wide array of interests, a constant desire to learn, and a wonderful soul.
+
+- Jurnell Cockhren: An amazingly wide and deeply learned individual inside and outside of technology. Working right at the line between software engineering, operations, and teaching, Jurnell's sharp mind, excellent sense of humor, and beautiful humanity endeared him to me from the beginning.
+
+- Sarah Guido: One of the people I greatly admire in the data science world, Sarah can take complex concepts break them down into understandable pieces with ease. Sarah's impressive engineering skills lead her to solve complex problems in data science and development alike.  With a great compassion to empower others, powerful logic, and writing, Sarah is truly an asset to our community.
+
+
+## CFP Opens Friday September 15th!
+PyTennessee 2018 is accepting all types of proposals starting Sept 15 through Nov 1, 2017. Due to the competitive nature of the selection process, we encourage prospective speakers to submit their proposals as early as possible. We’re looking forward to receiving your best proposals for tutorials and talks. Lightning talk sign-ups will be available at the registration desk the day of the event.
+
+Note: To submit a proposal, sign up or log in to your account and proceed to your account dashboard!
+
+## todo: confirm that this is true
+## Special Sponsorship Profile
+Intellovations has been the first sponsor of PyTN for all 5 years we have had the conference, and in addition to their financial support, they have been providing words of encouragement behind the scenes for all those years as well. They are truly part of the PyTN family, and for that, we are greatly appreciative and lucky to call them friends.
+
+## todo: get date for when Poung Coders begin
+## Young Coders
+We will be having young coders again this year. Tickets will be available December 15th at noon and they normally go fast.  The program will be lead by Brad Montgomery, a masterful python teacher, again this year. This free program is open to ages 12-17, and all attendees will receive take home books and a Chromebook.
```

Comment:
Capitalize "Young Coders" and "python".

And Young Coders tickets are on sale right now. I can take them off sale and we can set an "on sale" date for those, if you think that works better? Thoughts?